config/credentials.yml.enc

```diff
@@ -1 +0,0 @@
-MWFeMxLnDQtI2fflPd7/oebi4QGQ7jzrbxbAhssgnbAEuAOTyyfpu6LLLcxUBU+jzZWViVofVi0GDMVJQGSxoUTpf0rOog9Y1ORfTCzTwJ0SxYnZs2wzpCcJy/hQn19bZejqkmxuZS54Rz3bD67RW9LTnhsoyzcPU9Ky04KyEuNOl2JNSmkD+ZFU+rmdk1tW8itRCYNAWFKjoyGfoKoKPILB7IvBZldqK8jWB9kyzMji86Zsvz19yp/0flVCpHrIJUDhN/Lqc3B0NnaL9HEoFr4ytPUsJr7cgIf+Cpc+9JMINFKq1PysnYkwsVQ8pxc4jeuAUlfx/4C11XcVRqLiDjSXHcNvwb6R8cfRJs6eCIrG8GS9UbbQ0OIEKzOwWxdLM8GtKYHqJR9HLXmUYhFC1zY7IUsD6ZZukXNq--xHOAevqAn0ccbp1w--Y9XsnjW0EBUp3Yvl5nIc8w==
```

Comment:
This needs to stay here.  See https://guides.rubyonrails.org/security.html#custom-credentials

Reply:
Leaving it in breaks when trying to run the development server without the correct master.key. You also can't run `credentials:edit` without first deleting the file. As far as I can tell nothing in the app actually uses these credentials - why does it need to be here?

If these are the production configs, can I suggest using the `--environment` flag with `credentials:edit` so that when contributors try to run the server on their machine, it doesn't break instantly? Otherwise we need to add a "remove config/credentials.yml.enc" step to the setup process.

To be fair, I might be missing the point of rails credentials. It seems like the idea is that everyone on your team has the same key, and so does the production server, but since it's not committed to git, you can make your app open source and hey, everyone on your team has shared credentials through git, but the GP does not. Seems sensible! Except when someone in the GP wants to actually _use_ your app, like run it themselves, they can't. You don't even get a hint of the structure of what's in the credentials file, so you better hope that something in the README explains it. And even if it does, you have to delete it and recreate it with your own `master.key` and a new `credentials.yml.enc` file - which you then can't commit back to your own fork because then every PR has this conflict.

Am I missing something here? How am I supposed to _not_ delete this file and be able to work on this project?

Sorry for rambling, I'm just a bit confused by this feature. There might be a super obvious solution I'm not seeing.

Reply:
Fair comments.  I don't tend to use it much myself, but the secret-key-base is in there, the one Rails uses for a lot of stuff.

If you can remove it from this PR for now, I'll look into how we can best make it a non-issue for people trying to run the app themselves.

Reply:
Actually, it looks like I'm not even using that, and have a different one set through env vars.  So yeah let's go ahead and delete it.